var/spack/repos/builtin/packages/hepmc3/package.py

```diff
@@ -85,4 +85,7 @@ def cmake_args(self):
                     self.define("HEPMC3_CXX_STANDARD", spec["root"].variants["cxxstd"].value)
                 )
 
+        if "+protobuf" in spec:
+            args.append(self.define("HEPMC3_CXX_STANDARD", "14"))
```

Comment:
```suggestion
        if spec.satisfies("+protobuf"):
            args.append(self.define("HEPMC3_CXX_STANDARD", "14"))
```

It is now recommended to use `satisfies()` instead of `in` for checking spec attributes.